src/neural/network_blas.cc

```diff
@@ -16,9 +16,10 @@
  along with Leela Chess.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-#include "neural/CL/transforms.h"
-#include "neural/factory.h"
-#include "neural/network.h"
+#include "network.h"
```

Comment:
```
neural/network.h
neural/blas/blas.h
etc
```

Reply:
Is that necessary or a suggestion?

Reply:
I let you do that change printf to cerr.
I my humble opinion, printf is superior or iostream and I always use printf on purpose, not because I don't know better.
iostream is a disaster of an api.

Reply:
I'd say it's necessary (style guide is style guide..), and it's easy to do in this PR.
Other suggestions is fine to do in other PRs.